jina/parsers/peapods/pea.py

```diff
@@ -34,7 +34,13 @@ def mixin_pea_parser(parser):
 
     gp.add_argument('--expose-public', action='store_true', default=False,
                     help='If set, expose the public IP address to remote when necessary, by default it exposes'
-                         'private IP address, which only allows accessing under the same network/subnet')
+                         'private IP address, which only allows accessing under the same network/subnet. Important to '
+                         'set this to true when the Pea will receive input connections from remote Peas')
+
+    gp.add_argument('--force-pull-socket-from-remote', action='store_true', default=False,
```

Comment:
```suggestion
    gp.add_argument('--force-pull-socket-from-remote', action='store_true', default=False,
```

this has to be renamed

Reply:
what name would u suggest?

Reply:
`--force-pull-socket-from-remote` is too casual, we have to be more careful on the naming esp. before 1.0:
1. no need to add `--force` as it is a store_true action
2. `pull-socket` is not accurate, think about it, does it change the push-pull type if you turn on/off this option? no. what is change is the bind/connect: who is bind, who is connect.
3. `remote`? unclear, who is remote? remote to who?

Reply:
I would say force because given the current logic is kind of an exception.

Remote is always a relative concept trying to clarify in the comment.

However I agree is not a very good name but I could not come with a better name